students/RimleeT/lesson02/grid.py

```diff
@@ -0,0 +1,68 @@
+"""
+Part 1 : Write a function that draws a grid(fixed grid)
+"""
+plus = '+'
+minus = '-'*4
+line = '|'
+space = ' '*4
+horizontal = (plus + minus)*2 + plus
+vertical = (line + space)*2 + line
+print (horizontal)
+print (vertical)
+print (vertical)
+print (vertical)
+print (vertical)
+print (horizontal)
+print (vertical)
+print (vertical)
+print (vertical)
+print (vertical)
+print (horizontal)
+
+
+"""
+Part 2 : Write a function print_grid(n) that takes one integer argument and prints a grid just like before, BUT the size of the grid is given by the argument.
+"""
+def print_grid(n):
+	size_unit = (n-1)//2
+	plus = '+'
+	minus = '-'*size_unit
+	line = '|'
+	space = ' '*size_unit
+	print ((plus + minus)*2 + plus)
+	for i in range(0,size_unit):
```

Comment:
Overall, your grindprint looks great! Just a note, when using range and your initial value is 0, you do not need to specify it. range (0, 3) is equivalent to range(3)